files/build_templates/sonic_debian_extension.j2

```diff
@@ -151,6 +151,9 @@ sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install click-d
 # Install tabulate >= 0.8.1 via pip in order to support multi-line row output for sonic-utilities
 sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install tabulate==0.8.2
 
+#Install pyroute2==0.5.14 via pip in order to support get ip interface from kernal for sonic-utilities
+sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install pyroute2==0.5.14
```

Comment:
Prefer to add this dependency to the setup.py file in the sonic-utilities repo rather than installing explicitly here.

Reply:
@jleveque, Yes this is for 201911. Sorry I missed 201911 in the title